tests/homework/homework.page.ts

```diff
@@ -0,0 +1,18 @@
+import { expect, Locator, Page } from "@playwright/test";
+ 
+
+export class homework{
```

Comment:
Class name starts with Capital letter, naming could be improved to be representative of the page